python/cudf/cudf/_lib/parquet.pyx

```diff
@@ -878,11 +882,40 @@ cdef class ParquetReader:
         return df
 
     def read(self):
-        dfs = []
+        dfs = None
```

Comment:
I think this can be initialized to `self._read_chunk()` as libcudf will return a valid (maybe an empty chunk) for the first read after changes from #15757

Reply:
Good catch, made the change 👍